python/src/aac/plugins/validators/reference_fields/_validate_reference_fields.py

```diff
@@ -1,5 +1,6 @@
 import logging
 
+from aac.lang.constants import DEFINITION_FIELD_FIELDS, DEFINITION_FIELD_NAME, DEFINITION_FIELD_TYPE, PRIMITIVE_TYPE_REFERENCE
```

Comment:
When leveraging existing validators for help with this task, noticed that majority were making use of the constants (which was a recent effort to better expand), so updated ones that I had been referencing that were not.